public/js/updateService.js

```diff
@@ -0,0 +1,40 @@
+document.querySelector("#update").addEventListener("click",event=>{
+    event.preventDefault();
+    const editService = {
+        title :document.querySelector("#updateTitle").value,
+        Industry :document.querySelector("#updatedIndustry").value,
+        Description :document.querySelector("#updatedDescription").value,
+        Start :document.querySelector("#updatedSessionStart").value,
+        End :document.querySelector("#updatedSessionEnd").value,
+        Hourly_rate :document.querySelector("#updatedHourlyRate").value,
+        Max_bookings :document.querySelector("#updatedMaxBookings").value,
+    }
+    fetch((`/api/booking/:id`),{
+        method:"PUT",
+        body:JSON.stringify(editService),
+        headers:{
+            "Content-Type":"application/json"
+        }
+    }).then(res=>{
```

Comment:
Same deal about Async Await vs Promise chaining as well. We will also redirect the page on the backend so this is not needed but catching errors is.